### PR TITLE
[dom] Interim production lists based on merged PRs

### PR DESCRIPTION
**Pull request by kraushm**
To have the first modules be installed

### CODE REVIEW

File: jenkins-builds/7.0.UP00-19.03-mc [hunk 1]
```diff
@@ -1,2 +1,26 @@
  abcpy-0.5.1-CrayGNU-19.03-python3.eb               --set-default-module
- CMake-3.14.1.eb
```
Comment (lucamar) [inline jenkins-builds/7.0.UP00-19.03-mc]: @kraushm Why did you remove `CMake`?

Comment (kraushm) [reply jenkins-builds/7.0.UP00-19.03-mc]: It just moved a bit further down in the list.

**Comment by kraushm** [comment]
Please wait with merging, the Installpaths are still the old ones...my bad!

File: jenkins-builds/7.0.UP00-19.03-mc (hunk 1 above)
Comment (lucamar) [reply jenkins-builds/7.0.UP00-19.03-mc]: Sorry, I had overlooked that change

File: jenkins-builds/7.0.UP00-19.03-mc [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+ GSL-2.5-CrayGNU-19.03.eb                           --set-default-module
+ GSL-2.5-CrayCCE-19.03.eb
+ GSL-2.5-CrayIntel-19.03.eb
+ h5py-2.8.0-CrayGNU-18.08-python2-parallel.eb
+ h5py-2.8.0-CrayGNU-18.08-python2-serial.eb
+ h5py-2.8.0-CrayGNU-18.08-python3-parallel.eb       --set-default-module
+ h5py-2.8.0-CrayGNU-18.08-python3-serial.eb
+ PyExtensions-2.7.15.1-CrayGNU-19.03.eb
+ PyExtensions-3.6.5.1-CrayGNU-19.03.eb
+ IPM-2.0.6-CrayCCE-19.03.eb                                              --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
```
Comment (jgphpc) [inline jenkins-builds/7.0.UP00-19.03-mc]: mc

**Comment by kraushm** [comment]
Due to conflicts this PR will be re-submitted.
